tests/components/bluetooth/test_config_flow.py

```diff
@@ -64,27 +68,6 @@ async def test_async_step_integration_discovery(hass):
     assert len(mock_setup_entry.mock_calls) == 1
 
 
-async def test_async_step_integration_discovery_during_onboarding(hass):
```

Comment:
Why do we remove this test?

Reply:
Merge conflict resolution error.  Will fix as soon as I can.  Traveling today

Reply:
0abff6e6ef6969ce780b467034a2b347cd396eda

Thanks for catching that 👍